CONTRIBUTION.md

```diff
@@ -0,0 +1,101 @@
+- Search for existing similar issues
+
+If your question is still not resolved, you can create an issue and ask a question.  
+And also a template from GitHub issue will help you write good bug report.
+
+## How to contribute?
+### Report bug
+Please check the following before reporting a bug.
+
+- Look up ['Known Issues'](https://github.com/sqlitebrowser/sqlitebrowser/issues/1688)
+- Search for existing similar issues
+- Try in nightly build
+
+If a bug has not yet been repoted or has not been resolved, please follow the below to help members identify the bug.
+
+- Use a good title to identify the bug
+    - Ex) `Crash DB4S when try to reset the settings`
+- Detailed steps to reproduce the bug
+    - Ex) `File -> Preferences -> General -> Restore Defaults`
+- What you actually expected to do and why
+    - Ex) `DB4S settings are initalized normally.`
+- System environment, DB4S version, Loaded extensions
+    - Ex) `macOS Catalina Version 10.15.6(19G2021), DB4S Version 3.12.0, No Extensions`
+- Attach screenshots or screencasts related to the bug
+- Crash log (if the app is forced to close)
+    - If you have a long crash log, [Pastebin](https://pastebin.com/) can help
+- Project file or database file what used (**NOT** required)
+
+And also a template from GitHub issue will help you write good bug report.
+
+### Suggest enhancement
+Please check the following before suggest enhancement
+
+- Search for existing simillar issues
+
+And the please follow the below to help members identify the suggest.
+
+- Use a good title to identify the suggest
+- Describe the detail steps for the enhancement you want to propose
+- Attach simple example of the proposal, already applied projects or screenshots
+
+### Translation
+Project translation can be done through the ['Qt Linguist'](http://doc.qt.io/qt-5/qtlinguist-index.html) program.
+Languages that are already being translated can be translated by modifying the corresponding translation file,
+and translations for new languages can be translated by adding a new entry to the TRANSLATION section of the src/src.pro file and then rebuilding the project.
+
+#### DO NOT translate key names in shortcuts
+Keyboard shortcuts get automatically localized by Qt, and translating key names may indeed break them. They show up on the translation file only for the rare occasions that one has to redirect a shortcut to another one in a specific language. But even then, you should use the English key names to get this working. Theoretically, one could follow Qt conventions for translated key names, and they should work, but the risk to stumble upon Qt bugs or moving conventions is high, so it is better to not translate keys and only remap the shortcut. If you want to preserve the original key sequence (recommended), just leave the translation empty and finished.
+
+## Styleguide
+Styleguides help keep project's source code look good and consistent.  
```

Comment:
Replace "keep project's" with "keep the project's".
Replace "look" with "looking".